services/app-api/handlers/reports/fetch.ts

```diff
@@ -14,6 +14,7 @@ export const fetchReport = handler(async (event, _context) => {
 
   // get current report metadata
   const reportMetadataParams = {
+    // TODO: chain other report types
```

Comment:
Blocking: Looks like we still need to make the change in this file like in the others.

Reply:
I believe this one is out of scope since we currently don't have a way to determine the `reportType` on fetch, and I think we'd have to update the API calls for that to happen.